### PR TITLE
New thumbnail for "Datasets" example  by adjusting contrast

### PR DESCRIPTION
**Pull request by kolibril13**
I think inverting the color in the example "Datasets with 3 or more spatial dimensions" makes the cell membrane structure more visible and the thumbnail looks better as well.
image
EDIT: Here is the link:
https://output.circle-artifacts.com/output/job/36e91085-5da7-463c-8721-be3bda712e4a/artifacts/0/doc/build/html/auto_examples/index.html

### CODE REVIEW

**Comment by grlee77** [comment]
I agree that is more visible, but I think I would prefer adjusting via a percentile so we don't need to invert the contrast. The following based on clipping to a factor scaled up a bit from the 99th percentile looks reasonable:

```Python
# clip to improve visibility
p99 = np.percentile(img, q=99)
img = np.clip(img, 0, 1.25 * p99)
```

What would help more is having the thumbnail not be from the first frame of the data, but something like the 10th frame where the cells are more visible. I tried setting `animation_frame=10` in the plotly imshow call, but it did not work.

**Comment by grlee77** [comment]
maybe something like
```Python

# clip to improve visibility
p99 = np.percentile(img, q=99)
img = np.clip(img, 0, 1.25 * p99)

# omit some slices that are partially empty
img = img[5:26]
```

**Comment by grlee77** [comment]
Do you know if it is common in the cellular imaging literature to show images with reversed contrast? (my background is in medical imaging/radiology rather than microscopy).

**Comment by kolibril13** [comment]
> Do you know if it is common in the cellular imaging literature to show images with reversed contrast? (my background is in medical imaging/radiology rather than microscopy).

My background is in Micro CT and there we use mostely bright background and dark tissue structure. 
I did not work with microscopy data eigther , but I've just searched "3D fluorescence microscopy" in google schoolar and I get mixed results for the colormaps:
1. article: dark background
2. article: bright background
3. article: dark background
4. article: dark background

I also could not find any conventions by googling for it, but I think the idea with keeping the dark background and clipping the grey values to enhance the contrast + cropping the stack to the relevant area is a good idea.

**Comment by kolibril13** [comment]
New version:
https://output.circle-artifacts.com/output/job/612406e7-e230-4ffc-a9a0-f7ccad873b24/artifacts/0/doc/build/html/auto_examples/data/plot_3d.html#datasets-with-3-or-more-spatial-dimensions
I think the right image looks a bit too bright:
image
@grlee77 : Feel free to commit enhancements directly to this branch.

**Review by grlee77 — APPROVED** [review]

File: doc/examples/data/plot_3d.py [hunk 1]
```diff
@@ -17,10 +17,23 @@
...
-fig = px.imshow(img, facet_col=1, animation_frame=0,
-                binary_string=True, binary_format='jpg')
-fig.layout.annotations[0]['text'] = 'Cell membranes'
-fig.layout.annotations[1]['text'] = 'Nuclei'
+
+# omit some slices that are partially empty
+img = img[5:26]
+
+upper_limit = 1.5 * np.percentile(img, q=99)
+img = np.clip(img, upper_limit)
```
Comment (lagru) [inline doc/examples/data/plot_3d.py:28]:
```suggestion
img = np.clip(img, 0, upper_limit)
```
I think this is causing the [failed CI](https://github.com/scikit-image/scikit-image/runs/5725860988?check_suite_focus=true).

**Comment by lagru** [comment]
@kolibril13 thanks! I took the liberty to commit the change, because you suggested that to @grlee77 earlier.

**Comment by grlee77** [comment]
Thanks @lagru, not sure how I accidentally deleted the `, 0` from that expression.

I don't know why the direct link to the documentation artifact is broken here, but I was able to still reach it from the artifacts tab on the CircleCI run: https://output.circle-artifacts.com/output/job/cc9fa7f7-cb90-4875-8700-cbaf7ce36ae8/artifacts/0/doc/build/html/auto_examples/index.html

**Comment by lagru** [comment]
Thanks @kolibril13 and @grlee77!

**Pull request merged**